### PR TITLE
Feature/add router

### LINKED ISSUE #8: Add Router
Acceptance criteria
- [ ] When a user lands on the home screen, they see a header and a display of movie posters
- [ ] When a user clicks on a movie and the details page is rendered, the URL updates to reflect that movie’s unique ID as well
- [ ] The user can click the browser forward & back arrows to navigate
- [ ] The user can click the header to return to the home screen

### PR DESCRIPTION
**Pull request by rachelJensen**
### Description
Installed Router and added routes to movie pages and home page

#### Type of change
- [ ] Bug fix
- [X] New feature
- [ ] New feature that breaks previous code
- [ ] Refactor

#### How Has This Been Tested?
- [x] dev tools
- [x] run local:8080
- [ x] console.log()

#### Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

#### Additional Comments:
Close #8

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -48,11 +46,17 @@ class App extends Component {
   render() {
     return (
       <main>
-        <Header reload={this.reload} />
-        {!this.state.movieID &&  <MoviesContainer movies={this.state.movies} findMovie={this.findMovie} />}
-        {(this.state.movieID && !this.state.error) && <MovieDetailer movie={this.state.movie} />}
-        {(!this.state.movieID && this.state.error) && this.error500()}
+        <Header  reload={this.reload}/>
```
Comment (saberg1) [inline src/components/App/App.js:49]: Nice job with the route refactoring.

File: src/components/App/App.js [hunk 2]
```diff
@@ -48,11 +46,17 @@ class App extends Component {
   render() {
     return (
       <main>
-        <Header reload={this.reload} />
-        {!this.state.movieID &&  <MoviesContainer movies={this.state.movies} findMovie={this.findMovie} />}
-        {(this.state.movieID && !this.state.error) && <MovieDetailer movie={this.state.movie} />}
-        {(!this.state.movieID && this.state.error) && this.error500()}
+        <Header  reload={this.reload}/> 
+        <Route exact path='/' render={() => <MoviesContainer movies={this.state.movies} findMovie={this.findMovie} />} />
+        <Route exact path='/:id' render={() => <MovieDetailer movie={this.state.movie} />} />
```
Comment (sertmer) [inline src/components/App/App.js:51]: Clean refactoring!

File: src/components/App/App.js [hunk 3]
```diff
@@ -29,6 +26,7 @@ class App extends Component {
   findMovie = (id) => {
       this.setState({movieID: id})
       this.hanldeSingleMovie(id)
+      //this is where route id is defined?
     }
     
   hanldeSingleMovie = (id) => {
```
Comment (sertmer) [inline src/components/App/App.js:32]: Probably a good idea to do more with the Error in the singleMovie fetch than just console.log it - might be a good idea to set it in state

**Review by sertmer — COMMENTED** [review]
All in all this looks solid! I'd encourage you to use a `<Switch>` eventually, to handle any bad routes, but great work getting router in here!

**Pull request merged**